recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/dom0_vcpu_pin.sh

```diff
@@ -7,6 +7,6 @@
 # Default-Stop:
 ### END INIT INFO
 
-# Force all VCPUs of Domain-0 to only run on PCPUs from 4 to 7 (A53 cores)
+# Force all VCPUs of Domain-0 to only run on DOM0_ALLOWED_PCPUS PCPUs (A53 cores)
```

Comment:
After replacement comment will say `# Force all VCPUs of Domain-0 to only run on 4-7s (A53 cores)`

Reply:
Ah, sorry. Ignore this.